benchmarks/scale_1k_20k/ct_scale_1k_20k.yaml

```diff
@@ -0,0 +1,22 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
+region: us-west-2
+
+max_workers: 9999999
+
+head_node_type:
+    name: head_node
+    instance_type: r5dn.16xlarge # Network optimized.
```

Comment:
Is this the same instance type as the real workload btw? Or are we recommending people to use this type? It makes sense the head node uses the network optimized instance, but I'd like to make sure if it mimics the real workload in terms of compute spec.